docs/application/dotnet/guides/nui/flex-layout.md

```diff
@@ -110,8 +110,78 @@ flexLayout.WrapType = FlexLayout.FlexWrapType.NoWrap;
 layoutView.Layout = flexLayout;
 ```
 
+ The following properties affect the specified child view areas:
+- `FlexAlignmentSelf`
+-  `FlexPositionType`
+-  `FlexAspectRatio`
+-  `FlexBasis`
+-  `FlexShrink`
+-  `FlexGrow`
 
-## Related Information
+> [!NOTE]
+> The `FlexAlignmentSelf`, `FlexPositionType`, `FlexAspectRatio`, `FlexBasis`, `FlexShrink`, and `FlexGrow` properties are added in Tizen 6.0.
+
+**Table: Properties and descriptions**
+
+| Property               | Type            | Description  |
+| -----------------------| --------------- | ------------ |
+| `FlexAlignmentSelf`    | AlignmentType   | The alignment of the item along the cross axis when free space is available. |
+| `FlexPositionType`     | PositionType    | The position type defines how it is positioned within its parent. |
+| `FlexAspectRatio`      | float           | The ratio between the width and the height of an item. |
+| `FlexBasis`            | float           | The item's default size before any flex grow and flex shrink calculations are performed. |
+| `FlexShrink`           | float           | Specifies how to shrink children along the main axis in the case that the total size of the children overflows the size of the flex layout on the main axis. |
```

Comment:
```suggestion
| `FlexShrink`           | float           | Specifies how to shrink children along the main axis in case, the total size of the children overflows the size of the flex layout on the main axis. |